extensions/src/libadalang-helpers.ads

```diff
@@ -76,8 +76,15 @@ package Libadalang.Helpers is
    --  Try to create a unit provider out of ``Project``. If not possible, call
    --  ``Abort_App``.
 
+   function Command_Line_Event_Handler
+     (Exit_On_Missing_File : Boolean) return Event_Handler_Reference;
+   --  Create an event handler with default callbacks for command line
+   --  applications. ``Exit_On_Missing_File`` will determine the behavior when
+   --  encountering a missing dependency. If ``False``, a warning will be shown
```

Comment:
Could you adjust the doc to make it explicit that warnings/error messages are put on the standard output? (`Abort_App` writes to the standard error stream, so this will avoid potential confusion)

Reply:
I actually changed the event handler to print its message on stderr, which makes more sense IMO